jax_unirep/evotuning.py

```diff
@@ -249,7 +251,10 @@ def step(i, x, y, state):
 
     # Load and check that params have correct keys and shapes
     if params is None:
-        _, params = init_func(rng=rng, input_shape=(-1, 10))
+        if mlstm_size == 1900:
+            params = load_params()
+        else:
+            _, params = init_func(rng=rng, input_shape=(-1, 10))
```

Comment:
Minor nitpick, but I think the original is a bit more concise, and accomplishes the same thing. (Initialize with something, clobber over if another condition is met.)

Reply:
What I wanted to accomplish here is defaulting to the paper weights if someone doesn't specify `params` for an `mlstm_size` of 1900. But I'm also not 100% happy with this solution. Maybe instead of giving a lot of defaults, we just need to make it very clear in the documentation of how one can evotune starting with the weights from the paper.

Reply:
Hmmm yeah. I think I'm in favour of putting in the documentation how to load weights from the paper. The added complexity can creep in over time, leading to code duplication (that I tried my best in this PR to remove as well). 

When combined with the weights licensing issue, I can see how some teams (hint hint) might want to avoid using the pre-trained weights. From prior experience with things like hierarchical and local models, it seems to me that making it easy to train "local" weights is going to be a more pragmatically productive thing than training on entire large datasets.

Reply:
agreed!